api/src/main/proto/stellarstation/api/v1/antenna/antenna.proto

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2018 Infostellar, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+package stellarstation.api.v1.antenna;
+
+option go_package = "github.com/infostellarinc/go-stellarstation/api/v1/antenna";
+
+option java_multiple_files = true;
+option java_outer_classname = "AntennaProto";
+option java_package = "com.stellarstation.api.v1.antenna";
+
+// Messages for modeling physical antenna configuration in API messages.
```

Comment:
Sounds a bit weird with the word messages twice. Maybe just:

"Messages for modeling physical antenna configuration in the API."